### PR TITLE
coral2: expand MPI section

### PR DESCRIPTION
**Pull request by garlick**
Since I've had to debug MPI on coral2 lately I thought I'd do a little work on the Cray MPICH info in the coral2 guide.  This is probably a bit rough and I'm very open to suggestions on reverting my changes to the original, restructuring, or whatever, so let's consider it a WIP.

### CODE REVIEW

**Comment by garlick** [comment]
Just force pushed an editing pass.  Probably could use a review now.

File: tutorials/lab/coral2.rst [hunk 1]
```diff
@@ -9,75 +9,129 @@ El Capitan, and Frontier are similar in that they all use the
...
+- Add ``-o pmi=NAME[,NAME,...]`` to control which PMI implementations
+  are offered by the flux shell to jobs, e.g. ``simple``, ``cray-pals``,
+  ``pmix``).
+
+- Add ``-o verbose=2`` to request the shell to print tracing info
+  from the PMI implementations.
+
+- Launch ``flux-pmi`` as a parallel program to test PMI in isolation.
+
+.. code-block:: lua
```
Comment (vsoch) [inline tutorials/lab/coral2.rst:121]:
```suggestion
.. code-block:: console
```

Comment (garlick) [reply tutorials/lab/coral2.rst:121]: Does the lua tag cause a problem?  The lua syntax highlighting seems to be working in the preview...

Comment (vsoch) [reply tutorials/lab/coral2.rst:121]: But isn’t it bash or console?

Comment (vsoch) [reply tutorials/lab/coral2.rst:121]: I guess if you like it, there is no issue! Someone in the future might see it and change it though (if they think it’s not correct).

Comment (garlick) [reply tutorials/lab/coral2.rst:121]: Oops!  No no, you are correct, I was not zeroing in on the right code block.  Will fix!

**Comment by garlick** [comment]
Force pushed a slight improvement to the `flux pmi` example - adding `--label-io` so the ranks can be distinguished.

Mentioning @jameshcorbett since he's probably the right person to give this a review/approval.

**Review by jameshcorbett — APPROVED** [review]
Looks like a big improvement, thanks @garlick !

**Pull request merged**